src/main/kotlin/de/itemis/mps/gradle/modelcheck/Plugin.kt

```diff
@@ -91,6 +91,10 @@ open class ModelcheckMpsProjectPlugin : Plugin<Project> {
                 tasks.create("checkmodels", JavaExec::class.java) {
                     dependsOn(resolveMps)
                     args(args)
+                    if (extension.executable != null) {
```

Comment:
Why do you want to use a different java executable than the one currently on `JAVA_HOME`?

Reply:
As discussed briefly with @sergej-koscejev, instead of messing with the environment variable we wanted to force e.g. Java11 from the  client side. (See also MPS slack conversation)

Background: We have migrated mps-gradle-plugin to 2019.3 and want to use the model check in our project with this plug-in.
CI build of our project runs fine with Java 11.
But the model check still seems to use Java 8 and fails:
Exception in thread "main" java.lang.UnsupportedClassVersionError: jetbrains/mps/errors/item/IssueKindReportItem has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0
If I manually set the env.JAVA_HOME parameter to Java11, then we get quite far and the check runs through. I thought, that it should work without messing with environment variables. just like our client project.

But if just setting JAVA_HOME to Java11 is the way to go, we could do this as well.